metainfo.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this file exist in the upstream repository. you shouldn't have it here.

Reply:
also it's not even referenced here.

Reply:
I removed it, since I just found out I manage that file when building my project.